x-pack/plugins/reporting/public/management/mount_management_section.tsx

```diff
@@ -40,7 +40,7 @@ export async function mountManagementSection(
           }}
         >
           <InternalApiClientProvider apiClient={apiClient}>
-            <IlmPolicyStatusContextProvider>
+            <IlmPolicyStatusContextProvider statefulSettings={config.statefulSettings.enabled}>
```

Comment:
Note for self: The config is already passed to this part in reporting so the major change is adding the prop to IlmPolicyStatusContextProvider to then use the boolean in the hook.